### PR TITLE
More paper

### PR DESCRIPTION
**Pull request by gallais**
First front-to-back read so I'm touching things all over the place.

We need to stop adding references because we're going to go beyond the 2 pages limit. 😱

### CODE REVIEW

**Comment by jfdm** [comment]
Thanks!

**Comment by jfdm** [comment]
Sorry for taking over the PR with some more commits. Hopefully, I haven't borked any local changes.

I am happy to merge this, and then flip from draft to final.

File: paper/content/conclusion.tex [hunk 1]
```diff
@@ -23,24 +27,38 @@
...
+This is a hard problem:
+%
+One has to not only combine their semantics but also the remainder of the workbench tooling.
+%
 The \emph{Effects} library~\cite{DBLP:conf/icfp/Brady13} provides clues as to how we can compose \acp{dsl} horizontally but not vertically.
-\jfdm{I have no idea how to expand this properly with it obvious I am waffling.}
 
-With future engineering satisfying these missing criteria is permissible.
+
+We strenuously believe that With future engineering satisfying these missing criteria is possible.
```
Comment (gallais) [inline paper/content/conclusion.tex]:
```suggestion
We strenuously believe that with future engineering satisfying these missing criteria is possible.
```

**Comment by gallais** [comment]
Looks good! I've given up on `refine` because my idea was too naïve.
I'm happy to ship now.

**Pull request merged**